flop

```diff
@@ -67,28 +67,38 @@ EOF
                             echo 'Error: environment is greater than 5 characters'
                             exit 1
                         fi
-                        source $HOME/.nvm/nvm.sh
-                        nvm install v18
-                        nvm use v18
-                        echo "Building stac-server..."
-                        curl -L -f --no-progress-meter -o - "https://github.com/stac-utils/stac-server/archive/refs/tags/${STAC_SERVER_TAG}.tar.gz" | tar -xz
-                        cd "$STAC_SERVER_DIR"
-                        npm install
-                        BUILD_PRE_HOOK=true npm run build
-                        mkdir -p ../modules/stac-server/lambda/api
-                        cp dist/api/api.zip ../modules/stac-server/lambda/api/
-                        mkdir -p ../modules/stac-server/lambda/ingest
-                        cp dist/ingest/ingest.zip ../modules/stac-server/lambda/ingest/
-                        mkdir -p ../modules/stac-server/lambda/pre-hook
-                        cp dist/pre-hook/pre-hook.zip ../modules/stac-server/lambda/pre-hook/
-                        cd ..
-                        cd modules/stac-server/historical-ingest/lambda/
-                        pip install -r requirements.txt --target package
-                        cd package
-                        zip -r ../../lambda.zip .
-                        cd ../
-                        zip ../lambda.zip main.py
-                        cd ../../../../
+                        if
+                            cat $TFVARS_FILE | grep deploy_stac_server | grep "true"
+                        then
+                            echo 'deploy_stac_server is true, building stac-server dependencies'
+                            source $HOME/.nvm/nvm.sh
+                            nvm install v18
+                            nvm use v18
+                            echo "Building stac-server..."
+                            curl -L -f --no-progress-meter -o - "https://github.com/stac-utils/stac-server/archive/refs/tags/${STAC_SERVER_TAG}.tar.gz" | tar -xz
+                            cd "$STAC_SERVER_DIR"
+                            npm install
+                            BUILD_PRE_HOOK=true npm run build
+                            mkdir -p ../modules/stac-server/lambda/api
+                            cp dist/api/api.zip ../modules/stac-server/lambda/api/
+                            mkdir -p ../modules/stac-server/lambda/ingest
+                            cp dist/ingest/ingest.zip ../modules/stac-server/lambda/ingest/
+                            mkdir -p ../modules/stac-server/lambda/pre-hook
+                            cp dist/pre-hook/pre-hook.zip ../modules/stac-server/lambda/pre-hook/
+                            cd ..
+                        fi
+                        if
+                            cat $TFVARS_FILE | grep include_historical_ingest | grep "true" ||
```

Comment:
not to bloat this conditional logic, but I think we should bomb the script if `include_historical_ingest = false` and `include_ongoing_ingest = true`, as the collections are created as part of the historical lambda, and ongoing ingest will fail without building/deploying historical ingest. @MattBialas please correct me if I'm wrong.

And it still seems like we're only building the historical-ingest/lambda package, do we also need ongoing-ingest?

Otherwise, looking good 👍

Reply:
As I understand it, even if we do not use the historical ingest capability, it is the same historical ingest lambda that will create the collections for the ongoing ingest to work. @MattBialas could you confirm?

Reply:
That's right – the Historical Lambda is responsible for both creating the collections, and then _**(possibly)**_ doing a Historical Siphon. So if we're deploying LYOSTAC, we must package/deploy the Historical Lambda.

But regarding: **historical=false, ongoing=true**
I think that's still a valid use case. Someone might not want any historical data, but they do want new data.
(I don't know how _**common**_ that will be, but I think we should allow it)

Reply:
The only place we currently "use" the `include_historical_ingest` value is here:
https://github.com/Element84/filmdrop-aws-tf-modules/pull/10/files/97cb0679b4baefb4580ea845b1e1a0a02bf2a4c1..4bf65ad66a35841208f5f4a5c40232f32b824320#diff-5e9f92ae44bb95546e3aec8ec08674b66d091575a787e327271d0c6e8ee8343dR49